SpatialGDK/SpatialGDK.uplugin

```diff
@@ -37,11 +37,12 @@
       "Name": "SpatialGDKEditorCommandlet",
       "Type": "Editor",
       "LoadingPhase": "Default",
-      "WhitelistPlatforms": [ "Win64" ]
+      "WhitelistPlatforms": [ "Win64", "Mac" ]
```

Comment:
double check with @joshuahuburn @m-samiec  if that is wanted. I reverted that change in my pr as the toolbar doesn't currently work with Mac anyway, so we simply don't whitelist them for this platform. That also doesn't give mixed signals of whether we support the Unreal editor on mac or not

Reply:
I think we still keep the "SpatialGDKEditorCommandlet" module enabled for Mac as it's used by the command line to generate schema & snapshot, launch local deployment, and the game client. We have this "hacking" workflow enabled since 0.3.0, and some customers are depending on it for evaluation.

Reply:
This is just the commandlet (the command line tool used for generating schema/snapshots etc.), so this should be fine. And it has a dependency on the SpatialGDKEditor. The Toolbar etc. modules are still not whitelisted.

Reply:
Oh - yep - what @sang-tian said :)